src/definitions.ts

```diff
@@ -145,6 +145,16 @@ export interface IUserData {
   unset(key: string);
 }
 
+export interface ForgotPasswordDetails {
+  email?: string;
+}
+
+export interface ResetPasswordDetails {
```

Comment:
This should include the email previously sent with `ForgotPasswordDetails` as well.